apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx

```diff
@@ -1,59 +1,109 @@
-import { planDetails } from "../../../utils";
+import { useAuth } from "@/modules/auth/hooks/useAuth";
+import { useWorkspace } from "@/modules/workspace/hooks/useWorkspace";
+import * as Progress from "@radix-ui/react-progress";
+import { BillingCategoriesEnum, planDetails } from "../../../utils";
+
+type BillingValues = {
+    count: number;
+    limit: number;
+};
+
+type ParsedBilling = {
+    events: BillingValues;
+    persons: BillingValues;
+    responses: BillingValues;
+};
+
 export const Billing = () => {
+    const { organizations } = useAuth();
+    const { workspace } = useWorkspace();
+
+    const currentWorkspace = organizations.find((org) => {
+        return workspace?.id === org?.id;
+    });
+
+    const parsedBilling = JSON.parse(currentWorkspace?.billingUsage) as ParsedBilling;
+
+    const getBillingValues = (category: BillingCategoriesEnum): BillingValues => {
+        switch (category) {
+            case BillingCategoriesEnum.EVENTS:
+                return parsedBilling.events;
+            case BillingCategoriesEnum.PERSONS:
+                return parsedBilling.persons;
+            case BillingCategoriesEnum.RESPONSES:
+                return parsedBilling.responses;
+            default:
+                return { count: 0, limit: 0 };
+        }
+    };
+
     return (
         <div className="w-[675px] pt-10 text-intg-text-4">
             <div className="w-[515px]">
                 <h3 className="font-semibold">Manage Billing</h3>
                 <p className="text-sm">
-                    Manage your billing information and invoice. for question about billing Contact
-                    Integraflow@gmail.com or Github
+                    Manage your billing information and invoices. For questions about billing, contact
```

Comment:
Can you remove the `For questions about billing, contact...`.

Reply:
@afolasope Please don't forget to update this

Reply:
> @afolasope Please don't forget to update this

Done